paddleseg3d/models/unet3d.py

```diff
@@ -44,189 +45,174 @@ def __init__(self, in_channels, n_classes, base_n_filter=8):
 
         self.lrelu = nn.LeakyReLU()
         self.dropout3d = nn.Dropout3D(p=0.6)
-        self.upsacle = nn.Upsample(
-            scale_factor=2, mode='trilinear', data_format="NCDHW")
+        self.upsacle = nn.Upsample(scale_factor=2,
+                                   mode='trilinear',
+                                   data_format="NCDHW")
         self.softmax = nn.Softmax(axis=1)
 
-        self.conv3d_c1_1 = nn.Conv3D(
-            self.in_channels,
-            self.base_n_filter,
-            kernel_size=3,
-            stride=1,
-            padding=1,
-            bias_attr=False)
-        self.conv3d_c1_2 = nn.Conv3D(
-            self.base_n_filter,
-            self.base_n_filter,
-            kernel_size=3,
-            stride=1,
-            padding=1,
-            bias_attr=False)
+        self.conv3d_c1_1 = nn.Conv3D(self.in_channels,
+                                     self.base_n_filter,
+                                     kernel_size=3,
+                                     stride=1,
+                                     padding=1,
+                                     bias_attr=False)
+        self.conv3d_c1_2 = nn.Conv3D(self.base_n_filter,
+                                     self.base_n_filter,
+                                     kernel_size=3,
+                                     stride=1,
+                                     padding=1,
+                                     bias_attr=False)
```

Comment:
If possible, avoid these changes, it is a bit messy.

Reply:
I think this is done by pre-commit. Maybe I have misconfigured something. I thinkg I can disable it and only use formatter on new files I add.